sections/earn/ClaimTab/ClaimOrCloseFeeButton.tsx

```diff
@@ -0,0 +1,78 @@
+import { useRouter } from 'next/router';
+import { useTranslation } from 'react-i18next';
+import { StyledButton } from '../common';
+import { Tooltip } from 'styles/common';
+import styled from 'styled-components';
+import Wei from '@synthetixio/wei';
+import ROUTES from 'constants/routes';
+
+const ClaimOrCloseFeeButton: React.FC<{
+	handleClaim: () => void;
+	handleCloseFeePeriod: () => void;
+	canClaim: boolean;
+	isBelowCRatio: boolean;
+	hasVoted: boolean;
+	hasClaimed: boolean;
+	totalRewards: Wei;
+	isCloseFeePeriodEnabled: boolean;
+}> = ({
+	handleClaim,
+	handleCloseFeePeriod,
+	canClaim,
+	isBelowCRatio,
+	hasClaimed,
+	isCloseFeePeriodEnabled,
+	totalRewards,
+	hasVoted,
+}) => {
+	const { t } = useTranslation();
+	const router = useRouter();
+	if (isCloseFeePeriodEnabled) {
+		return (
+			<PaddedButton variant="primary" onClick={handleCloseFeePeriod}>
+				{t('earn.actions.claim.close-fee-period')}
+			</PaddedButton>
+		);
+	}
+	return (
+		<Tooltip
+			hideOnClick={true}
+			arrow={true}
+			placement="bottom"
+			content={t('earn.actions.claim.ratio-notice')}
+			disabled={!canClaim || !isBelowCRatio}
+		>
+			<PaddedButtonContainer>
+				{hasVoted ? (
```

Comment:
So if `hasVoted` is true, we show the `not voted` button?

Reply:
Also can you please check with @andytcf if this needs to be disabled for L2?

Reply:
> So if hasVoted is true, we show the not voted button?
Nice catch, that looks like a mistake
> Also can you please check with @andytcf if this needs to be disabled for L2?
Will chat to him